config/build/babelrc.js

```diff
@@ -16,8 +15,12 @@ import { vitaminResolve } from '../utils';
 export default (env, dev) => ({
     // order is: last to first
     presets: [
-        env === 'client' ? [presetLatest, { es2015: { modules: false } }]
-            : [presetEnv, { modules: false, targets: { node: 'current' }, useBuiltIns: true }],
+        [presetEnv, {
+            modules: false,
+            useBuiltIns: true,
+            targets: env !== 'client' ? { node: 'current' }
+                : { uglify: !dev, browsers: ['not ie < 9'] },
```

Comment:
It's an option that says we use uglifyjs, because uglifyjs doesn't support ES6.
https://github.com/babel/babel-preset-env#targetsuglify